Source/SpatialGDK/Private/SpatialGameInstance.cpp

```diff
@@ -62,8 +62,9 @@ bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
 		GetEngine()->ShutdownWorldNetDriver(GetWorldContext()->World());
 	}
 
-	FURL BaseURL = WorldContext->LastURL;
-	FURL URL(&BaseURL, TEXT("127.0.0.1"), ETravelType::TRAVEL_Absolute);
+	// This will use the URL / Map that was setup by the server worker (which is loaded first).
```

Comment:
Comment inaccurate, it will take either PIE map or the commandline map or the default map (in that order, I believe).